docs/VnVPlan/VnVPlan.tex

```diff
@@ -216,57 +216,616 @@ \section{System Test Description}
+  
+      How test will be performed: The function that accesses the micro-controller will be run using the appropriate inputs.
+      After it has ran we will check to see if the position, colour, and type of piece are stored in the micro-controller.  
+  
+      \item{NB-2\\}
+  
+      Type: Functional, Dynamic, Manual
+                        
+      Initial State: The game is in Normal Mode.
+                          
+      Input: The user will hold down the Resign button for ENDTIME seconds.
+                          
+      Output: The game state will change to the Game Inactive State.
+                          
+      Test Case Derivation: The game shall be in the Game Inactive State due to the Resign button being pressed.
+      
+      How test will be performed: The function that changes the game state will be run using the appropriate inputs.
+      After it has ran we will check to see if the game state is in the Game Inactive State.
+  
+      \item{NB-3\\}
+  
+      Type: Functional, Dynamic, Manual
+                        
+      Initial State: The game is in Normal Mode.
+                          
+      Input: Both users will hold down the Draw button for ENDTIME seconds each located on their side of the board.
+                          
+      Output: The game state will change to the Game Inactive State.
+                          
+      Test Case Derivation: The game shall be in the Game Inactive State due to the Draw buttons being pressed.
+      
+      How test will be performed: The function that changes the game state will be run using the appropriate inputs.
+      After it has ran we will check to see if the game state is in the Game Inactive State.
+  
+      \item{ND-1\\}
+  
+      Type: Functional, Dynamic, Manual
+                        
+      Initial State: The game is in Normal Mode.
+                          
+      Input: The system shall send the micro-controller output over the chosen data transfer
+      method as an input to the web application.
+                          
+      Output: The web application shall receive the micro-controller output sent over the chosen data transfer
+      method.
+                          
+      Test Case Derivation: The web application should receive new information regarding the board state over the chosen data transfer method. 
+  
+      How test will be performed: The function that checks to see if any information is being transmitted from the micro-controller
+      will be run using the appropriate inputs. After it has ran we will check to see if the appropriate information has been received.
+  
+      \item{NA-2\\}
+  
+      Type: Functional, Dynamic, Manual
+                        
+      Initial State: The game is in Normal Mode.
+                          
+      Input: The web application shall receive the micro-controller output sent over the chosen data transfer
```

Comment:
Just say Bluetooth instead of chosen data transfer method since we know what data transfer method we're using.